app/client/jsx/JitsiVideo.jsx

```diff
@@ -73,7 +73,7 @@ class JitsiVideo extends Component {
                     SHOW_JITSI_WATERMARK: false,
                     DEFAULT_REMOTE_DISPLAY_NAME: 'Fellow Clarendonite',
                     SHOW_WATERMARK_FOR_GUESTS: false,
-                    TOOLBAR_BUTTONS: this.toolbarButtons,
+                    TOOLBAR_BUTTONS: toolbarButtons,
```

Comment:
did you remove this in previous PR? oops

Reply:
hm no, I think you may be readin the PR wrong

Reply:
hm no, I think you may be readin the PR wrong